maltd-frontend/cypress/integration/tests/find-users.js

```diff
@@ -0,0 +1,43 @@
+/* global cy, before */
+// / <reference types="Cypress" />
+
+import LandingPage from "../../support/pageObjects/landing-page";
+
+describe("The landing pages tests", () => {
+  // Runs once before all tests in the block
+  before(() => {
+    cy.fixture("userData.json").as("users");
+  });
+
+  it("Finds users", () => {
+    // Launches the url
+    cy.visit("/");
+
+    // Validates the Find button is disabled by default
+    LandingPage.getFindButton().should("be.disabled");
+
+    cy.get("@users").then(users => {
```

Comment:
you got this working with `cy.get` - nice! what was the issue yesterday?

Reply:
It seemed like a Cypress syntax mistake. Started from scratch and worked fine!